### ISSUE TITLE
multiple panelid's give error

### ISSUE THREAD

**Issue by p0nt** [Mikesch-mp/icingaweb2-module-grafana#29]
Thank you for writing this module, its awesome!

One error that I get when using multiple panel id's:

```
sprintf(): Too few arguments

#0 [internal function]: Icinga\Application\ApplicationBootstrap->Icinga\Application\{closure}(2, 'sprintf(): Too ...', '/usr/share/icin...', 301, Array)
#1 /usr/share/icingaweb2/modules/grafana/library/Grafana/ProvidedHook/Grapher.php(301): sprintf('<a href="http:/...', 'http', 'icinga.xsnews.i...', 'db', 'icinga2-dfeedin...', 'abp003', 'dfeedinfo-infee...', '6h', '<img src="http:...')
#2 zend.view:///usr/share/icingaweb2/modules/monitoring/application/views/scripts/show/components/grapher.phtml(2): Icinga\Module\Grafana\ProvidedHook\Grapher->getPreviewHtml(Object(Icinga\Module\Monitoring\Object\Service))
#3 /usr/share/php/Icinga/Web/View.php(231): include('zend.view:///us...')
#4 /usr/share/icingaweb2/library/vendor/Zend/View/Abstract.php(877): Icinga\Web\View->_run('/usr/share/icin...')
#5 zend.view:///usr/share/icingaweb2/modules/monitoring/application/views/scripts/partials/object/detail-content.phtml(3): Zend_View_Abstract->render('show/components...')
#6 /usr/share/php/Icinga/Web/View.php(231): include('zend.view:///us...')
#7 /usr/share/icingaweb2/library/vendor/Zend/View/Abstract.php(877): Icinga\Web\View->_run('/usr/share/icin...')
#8 zend.view:///usr/share/icingaweb2/modules/monitoring/application/views/scripts/service/show.phtml(25): Zend_View_Abstract->render('partials/object...')
#9 /usr/share/php/Icinga/Web/View.php(231): include('zend.view:///us...')
#10 /usr/share/icingaweb2/library/vendor/Zend/View/Abstract.php(877): Icinga\Web\View->_run('/usr/share/icin...')
#11 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action/Helper/ViewRenderer.php(904): Zend_View_Abstract->render('service/show.ph...')
#12 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action/Helper/ViewRenderer.php(925): Zend_Controller_Action_Helper_ViewRenderer->renderScript('service/show.ph...', NULL)
#13 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action/Helper/ViewRenderer.php(964): Zend_Controller_Action_Helper_ViewRenderer->render()
#14 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action/HelperBroker.php(272): Zend_Controller_Action_Helper_ViewRenderer->postDispatch()
#15 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action.php(518): Zend_Controller_Action_HelperBroker->notifyPostDispatch()
#16 /usr/share/php/Icinga/Web/Controller/Dispatcher.php(76): Zend_Controller_Action->dispatch('showAction')
#17 /usr/share/icingaweb2/library/vendor/Zend/Controller/Front.php(937): Icinga\Web\Controller\Dispatcher->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response))
#18 /usr/share/php/Icinga/Application/Web.php(389): Zend_Controller_Front->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response))
#19 /usr/share/php/Icinga/Application/webrouter.php(109): Icinga\Application\Web->dispatch()
#20 /usr/share/icingaweb2/public/index.php(4): require_once('/usr/share/php/...')
#21 {main}
```

**Comment by Mikesch-mp** [comment]
Found the problem, will release fix soon.

**Comment by Mikesch-mp** [comment]
Fixed in v1.1.1. Please download and install this version.

**Comment by p0nt** [comment]
tested, can confirm it works.
